src/responder.js

```diff
@@ -219,6 +219,37 @@ export default class Responder {
     );
   }
 
+  resteem(
+    targetUsername = this.targetUsername,
+    targetPermlink = this.targetPermlink
+  ) {
+    this._throwErrorIfNoKey();
+    this._throwErrorIfNoPermlink(targetPermlink);
+
+    const wif = this.postingKey || this.activeKey;
+
+    const action = JSON.stringify(['reblog'], {
+      account: this.responserUsername,
+      author: targetUsername,
+      permlink: targetPermlink
+    })
+
+    return steem.broadcast.customJsonAsync(
+      wif,
+      [],
+      [this.responderUsername],
+      'follow',
```

Comment:
Why the id for transaction is "follow" ? I don't remember exactly but I think it should be a random string like a uuid. since the [new commit](https://github.com/steemit/steem/commit/3f453bbb65f14a528072fe4efde964dbb5cf09b5) in node from the second execution it returns this error:
```
Unhandled rejection RPCError: Assert Exception:_callbacks.find( txid ) == _callbacks.end(): Transaction is a duplicate
    at new RPCError (/Users/p0o/dev/steem-bot-tests/steem-bot/node_modules/steem/lib/api/transports/http.js:39:106)
    at /Users/p0o/dev/steem-bot-tests/steem-bot/node_modules/steem/lib/api/transports/http.js:74:13
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```